### PR TITLE
fix: disabled cursor

### LINKED ISSUE #19401: [style] wrapper cursor style not corectlly on disabled checkbox
- [ x ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/xz398v5y1w)

### Steps to reproduce
Move mouse slowly between disabled checkbox and checkbox-wrapper

bug

### What is expected?
cursor is always 'not-allowed'.

### What is actually happening?
cursor is always 'pointer' when mouse on wrapper.

| Environment | Info |
|---|---|
| antd | 3.24.2 |
| React | latest |
| System | mac os 10.14.4 |
| Browser | chrome 78 |

### PR DESCRIPTION
**Pull request by chiaweilee**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ x ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

fix #19401

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ x ] Doc is updated/provided or not needed
- [ x ] Demo is updated/provided or not needed
- [ x ] TypeScript definition is updated/provided or not needed
- [ x ] Changelog is provided or not needed

### CODE REVIEW

File: components/checkbox/style/mixin.less [hunk 1]
```diff
@@ -143,6 +143,10 @@
     .@{checkbox-prefix-cls}-wrapper:hover &::after {
       visibility: hidden;
     }
+
+    .@{checkbox-prefix-cls}-wrapper {
```
Comment (yoyo837) [inline components/checkbox/style/mixin.less]: 缺 `&` 吧，同一个元素

Comment (yoyo837) [reply components/checkbox/style/mixin.less]: 不对，是上一级

**Review by yoyo837 — CHANGES_REQUESTED** [review]
`.@{checkbox-prefix-cls}-disabled` 是 `.@{checkbox-prefix-cls}-wrapper` 的后代元素，另有 `.@{checkbox-prefix-cls}-wrapper-disabled` 元素。

**Review by yoyo837 — APPROVED** [review]

**Comment by yoyo837** [comment]
@afc163 `netlify` 怎么重新触发部署？

**Comment by yoyo837** [comment]
好像所有PR的 `netlify` 都这样了...

**Comment by afc163** [comment]
> 好像所有PR的 netlify 都这样了...

netlify service down.

**Pull request merged**